nussl/separation/duet.py

```diff
@@ -71,15 +76,24 @@ def __init__(self, input_audio_signal, num_sources,
 
         self.attenuation_delay_histogram = None
         self.non_normalized_hist = None
-        self.smoothed_hist = None
         self.peak_indices = None
+        self.p = p
```

Comment:
If you can't find a better name for these in the paper, then add a comment so that it's clear what they do. Or make up a descriptive name.